code/game/mecha/equipment/tools/medical_tools.dm

```diff
@@ -401,6 +402,13 @@
 		create_reagents(max_volume)
 		synth = new (list(src),0)
 
+	Destroy()
+		syringes = null
```

Comment:
setting these vars null may not be adequate. For single object references, this is why we have QDEL_NULL

no such macro exists for lists yet (maybe you should add one) so for a list you have to loop through and qdel each thing in it, then set the list equal to list()

Reply:
a macro does exist, QDEL_NULL_LIST, i used that instead of adding a new one, although it's different from /tg/ and I'm not sure which one is better.